docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -188,3 +188,40 @@ POST _ml/set_upgrade_mode?enabled=false&timeout=10m
 DELETE temp_ml_config
 --------------------------------------------------
 // TEST[skip:TBD]
+
+[discrete]
+[[ml-debian8-memory]]
+== Suboptimal job assignment on Debian 8
+
+Where possible {ml} jobs are assigned to nodes based on the memory requirement
+of the job and the memory available on the node. This breaks down when the
+amount of memory on a node cannot be accurately determined, and then jobs are
+assigned simply by balancing numbers of jobs per {ml} node. This could lead to
+a situation where all the big jobs are on one node and all the small jobs on
+another.
```

Comment:
Nit. Please feel free to ignore my suggestion if you think it does not fit. I wanted to specify a bit more accurately what "big" and "small" mean. 

```suggestion
of the job and the memory available on the node. However, in certain cases, the
amount of memory on a node cannot be accurately determined, and then jobs are
assigned by balancing numbers of jobs per {ml} node. It may lead to
a situation where all the jobs with high memory requirements are on one node and the less memory-intensive jobs on
another.
```